tests/unit/test_data_callbacks.py

```diff
@@ -41,32 +42,80 @@ def __init__(self):
                              'PSOrthoTile',
                              'REScene'
                          ])
-def test_item_types_success(item_types):
+def test_item_types_success_data(item_types):
     ctx = MockContext()
-    result = check_item_types(ctx, 'item_types', [item_types])
+    result = data.check_item_types(ctx, 'item_types', [item_types])
     assert result == [item_types]
 
 
-def test_item_types_fail():
+def test_item_types_fail_data():
     ctx = MockContext()
     with pytest.raises(click.BadParameter):
-        check_item_types(ctx, 'item_types', "bad_item_type")
+        data.check_item_types(ctx, 'item_types', "bad_item_type")
 
 
-def test_item_type_success():
+def test_item_type_success_data():
     ctx = MockContext()
     item_type = "PSScene"
-    result = check_item_type(ctx, 'item_type', item_type)
+    result = data.check_item_type(ctx, 'item_type', item_type)
     assert result == item_type
 
 
-def test_item_type_fail():
+def test_item_type_fail_data():
     ctx = MockContext()
     with pytest.raises(click.BadParameter):
-        check_item_type(ctx, 'item_type', "bad_item_type")
+        data.check_item_type(ctx, 'item_type', "bad_item_type")
 
 
-def test_item_type_too_many_item_types():
+def test_item_type_too_many_item_types_data():
     ctx = MockContext()
     with pytest.raises(click.BadParameter):
-        check_item_types(ctx, 'item_type', "PSScene,SkySatScene")
+        data.check_item_types(ctx, 'item_type', "PSScene,SkySatScene")
+
+
+# Identical tests to above, but for subscriptions CLI
+@pytest.mark.parametrize("item_types",
```

Comment:
This parameterization doesn't seem like it will exercise the code in the data cli differently. It more seems like it's testing the behavior of the specs module, which is already tested in another unit test. I am not seeing the value-add, where-as the cost is that we have to update this test if the spec ever changes. Unless I'm missing something, I recommend going back to one item type, probably PSScene is the best.

Reply:
Ah, yeah you're right. Let's change to a single item type, PSScene, for maintainability!